scipy/stats/morestats.py

```diff
@@ -1083,6 +1100,23 @@ def boxcox_normmax(x, brack=(-2.0, 2.0), method='pearsonr'):
         'all'
             Use all optimization methods available, and return all results.
             Useful to compare different methods.
+    optimizer : callable, optional
+        If `lmbda` is None, `optimizer` is the scalar optimizer used to find
+        the value of `lmbda` that minimizes the negative log-likelihood
+        function. `optimizer` is a callable that accepts one argument:
+
+        fun : callable
+            The objective function, which evaluates the negative
+            log-likelihood function at a provided value of `lmbda`
```

Comment:
@tupui you promted this: `lmbda` should not be written with single-backticks (and thus rendered in italics) because it is not a parameter of this function.

However, `lmbda` is defined above:

![image](https://user-images.githubusercontent.com/6570539/114433640-e7a92400-9b76-11eb-93c9-a38f9b066c24.png)

What do you think about changing it to double-backticks here to match that? 

We could also consider renaming the output of this function from `maxlog` to `lmbda`. What do you think of that, @mloning?

Reply:
Yes but I got confused here. Because then what I understand is that I have no control over the value of `lmbda` and then if it is for some reason `None` the `func` will not be used.

Reply:
> then if it is for some reason None the func will not be used.

I see you just eliminated those parts, so hopefully that is not contributing to the confusion anymore?

Reply:
With you suggestions it is better yes.